### ISSUE TITLE
Input readonly does not meet contrast requirements

### ISSUE THREAD

**Issue by eirikbacker** [digdir/designsystemet#2604]
Jeg får at kontrasten er `3.56:1` mellom `color/neutral/border-default` og `color/neutral/background-subtle`. Burde kanskje check og dot (også for switch) bare være `color/neutral/text-default` da slik som på input? :relaxed:
